operations/jsonnet/enterprise/main.libsonnet

```diff
@@ -64,7 +64,7 @@ tempo {
       storage: {
         type: $._config.backend,
         s3: {
-          bucket: $._config.bucket + '-admin',
+          bucket_name $._config.bucket + '-admin',
```

Comment:
are we missing a colon here?

Reply:
Fix the property and miss out the colon 🤦 Must have stuffed up whilst commiting, thanks Annanay. :)
Have also signed the CLA!